src/nvim/ex_cmds.c

```diff
@@ -5920,23 +5934,35 @@ static void sign_undefine(sign_T *sp, sign_T *sp_prev)
 
 /*
  * Get highlighting attribute for sign "typenr".
- * If "line" is TRUE: line highl, if FALSE: text highl.
+ * Gets the highlight corresponding to "type", which should be
+ * SIGN_TEXT, SIGN_LINEHL or SIGN_NUMHL.
```

Comment:
One advantage of an enum is that it removes the need to list such things explicitly in docstrings. And if this only accepts a subset that should be checked with assert() or abort() instead.